custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -337,6 +346,8 @@ def _check_artifacts_and_get_run_language(self):
                 artifact_language = RunLanguage.JAVA
             elif len(julia_artifacts) > 0:
                 artifact_language = RunLanguage.JULIA
+            elif len(triton_inference_server_artifacts) > 0:
+                artifact_language = RunLanguage.TRITON_ONNX
```

Comment:
I'm going to add more backends here, depending on the `config.pbtxt` values set for `platform` and `backend`.
All the Triton backends will be mapped to the same `TritonPredictor`.

Reply:
I wonder if we should finish with this language/artifact guessing and just require user to provide a proper parameter.

Reply:
Agree, I added a boolean option `--with-triton-server` which maps to the `RuntimeLanguage.OTHER`.
Since Triton models can have `*.onnx`, `*.py` and `*.pbtxt` files and it will fail validation here.